dependency-check-suppressions.xml

```diff
@@ -20,6 +20,11 @@
         <cve>CVE-2019-16869</cve>
     </suppress>
 
+    <suppress>
+        <cve>CVE-2020-11612</cve>
+        <cve>CVE-2020-7009</cve>
+    </suppress>
```

Comment:
Please add notes what libs and versions these are for, so they can be removed when libs get updated.